src/server.ts

```diff
@@ -53,10 +53,12 @@ app.get('/', (_req, res) => {
   res.redirect('/dashboard');
 });
 
+type DashboardScene = Pick<Scene, "id" | "name" | "imageUrl">
+
 app.get('/dashboard', (_req, res) => {
   const promiseRooms = HueAPI.getGroups().then(groups => {
     const rooms = Dashboard.getRoomsFromGroups(groups);
-    return rooms;
+    return rooms.map(room => ({...room, scenes: [] as DashboardScene[]}));
```

Comment:
Oh, also a bug fix, for the case of a scene-less room. Initialising "scenes" here means that every room ends up with a scenes array, even if it's empty.